src/containers/SecretsModal/SecretsModal.js

```diff
@@ -228,26 +212,29 @@ export /* istanbul ignore next */ class SecretsModal extends Component {
     const stateValue = e.selectedItem.id;
```

Comment:
In this handler, for any annotation value that is blank or one of the docker or git default text, it will go ahead and change those value to the default one of the new selected access value (does not affect unique values). Which means that if it was previously invalid (because it was left blank) then it also removes this invalid designation since it changed it to a correct default value. That being said, I could not see this behavior when I pulled the PR. 


Example: 
Here we see that each field has a default git access value.
<img width="752" alt="Screen Shot 2019-11-06 at 4 38 03 PM" src="https://user-images.githubusercontent.com/49996607/68340168-e804dc00-00b3-11ea-9e82-4b6f8968e446.png">

Now, in this one I delete the second annotation value and we can see that it marks it as invalid because it is blank.
<img width="752" alt="Screen Shot 2019-11-06 at 4 38 12 PM" src="https://user-images.githubusercontent.com/49996607/68340212-f9e67f00-00b3-11ea-9ab7-c0a85f0a24a9.png">


Finally, I modified the last annotation value to be a unique one and then went ahead and change the `Access To` value from `Git` to `Docker`.  We can now see how after this, the second annotation value still is marked as invalid although it has a default docker access to value.
<img width="750" alt="Screen Shot 2019-11-06 at 4 38 26 PM" src="https://user-images.githubusercontent.com/49996607/68340252-0ec31280-00b4-11ea-9060-5f4d190f98bf.png">

Reply:
Good catch @carlos-logro! The latest code I pushed fixes this error if you want to give it another try 👍